src/esp32cam.h

```diff
@@ -29,33 +42,194 @@ class CameraClass
   bool
   changeResolution(const Resolution& resolution, int sleepFor = 500);
 
+  /** \brief Change camera resolution.
+   *  \pre Change Camera Frequence Clock
+   *  \param iMhz new Frequence, must be in MHZ
+   */
+  bool
+  changeFrequence(int iMhz);
+
+  /** \brief Change Sensor Contrast level
+  * must be -2 to +2, default 0
+  */
+  bool
+  changeContrast(int ilevel);
+
+  /** \brief Change Brightness level
+   *  \param ilevel must be -2 to +2, default 0
+   */
+  bool
+  changeBrightness(int ilevel);
+
+  /** \brief Change Saturation level
+   *  \param ilevel must be -2 to +2, default 0
+   */
+  bool 
+  changeSaturation(int ilevel);
+
+  /** \brief Change SpecialEffect
+   *  \param ilevel must be > 0 && <= NUM_SPECIAL_EFFECTS, default 0
+   *  \	1 = no effect
+   *  \ 2 = negative
+   *  \ 3 = black and white
+   *  \ 4 = reddish
+   *  \ 5 = greenish
+   *  \ 6 = blue
+   *  \ 7 = retro
+   */
+  bool
+  changeSpecialEffect(int ilevel);
+
+  /** \brief Change WbMode
+   *  \param ilevel must be > 0 && <= NUM_WB_MODES, default 0
+   *  \ 0 = default
+   *  \ 1 = sunny
+   *  \ 2 = cloudy
+   *  \ 3 = office
+   *  \ 4 = home
+   */
+  bool
+  changeWbMode(int ilevel);
+
+  /** \brief Change AE Levels
+   *  \param ilevel must be -2 to +2, default 0
+   */
+  bool
+  changeAeLevels(int ilevel);
+  
+  /** \brief Enable/Disable  AEC (Auto Exposure Control)
+   *  \param iEnable must be 0(disable) or 1 (enable)
+   */
+  bool
+  changAutoExposurecontrol(int iEnable);
```

Comment:
Some drive-by comments while I was looking at OV2640 libraries...

There are a bunch of methods named `chang...`. I think you mean `change...`